docs/blog/content/material/proxy.en.md

```diff
@@ -1,7 +1,295 @@
+
+Suppose that a database is set to max_user_connections=80, and it is routed to 100 tables resulting in being impossible to create 100 connections in the same time and no merging result to return. 
+
+In order to resolve the problem above, CONNECTION_STRICTLY comes out, it doesn't make use of ResultSet flow which causes memory increasing. CONNECTION_STRICTLY doesn't need to keep in touch with database, it will release connection after retrieving all the data in ResultSet.
+
+We use the same setting max_user_connections=80 as example, the database is routed to 100 tables. Proxy create 80 connection for quering data at first, the other 20 connections will be cached into pool. These 20 connections will be created successfully one after another within quering completion before.
+
+If you feel confused, please keep it in mind CONNECTION_STRICTLY is a scenario that max_user_connections is less than maximums of tables routed to.
+
+### 03 Summary
+Sharding-Sphere has been continuous improvement and development since 2016. A growing number of componies and individuals adaop it, and they provide many successful cases for us.  We will move forward to impove current features, achieve soft transaction, data management and so on in succession. If someone has good ideas or wants to do proposals, welcome to join Sharding-Sphere open source project.
+
+*   https://github.com/sharding-sphere/sharding-sphere/
+    
+*   https://gitee.com/sharding-sphere/sharding-sphere/
+    
+
+  
+
+  
+
+### Q&A
+
+Q1: What's Sidecar？
+
+A1: Sharding-Sidecar is the third product of Sharding-Sphere, it's on the way. It's in form of DaemonSet to agent all the databases targeted at cloud native database proxy.
+
+  
+
+Q2: Is it correct that "please keep it in mind CONNECTION_STRICTLY is a scenario that max_user_connections is less than maximums of tables routed to"？
+
+A2: CONNECTION_STRICTLY is for decreasing connections. It's small for max_user_connections, so we use CONNECTION_STRICTLY mode.
+
+  
+
+Q3: Within the "stmt.setFetchSize(custom_size)" scenario, it's similar with ORM framworks like Mybatis to query a amount of datas into memory, these data are stored into list in general to handle them. They still occupy memory.
+
+A3: It's client memory in this situation, it is no influence of Proxy.
+
+  
+
+Q4: If we have lots of data in memory, is it only one way to use native JDBC to query/handle datas every time without putting them into memory？
+
+A4: Client takes control of Mybatis, it has nothing to do with Proxy.
+
+  
+
+### Link of Live playback
+
+https://m.qlchat.com/topic/details?topicId=2000001395952730&minimal=1
+
+Do you want to know more about Sharding-Sphere？
+
+**Come to "2018 DAMSChina Data Asset Management Summit"**
+
+**Attend the analysis of lecturer Zhang Liang who is responsible for database development of JD Finance**
+
```

Comment:
May be missing the last picture.

Reply:
After Communicating with Trista, there is no need to add the picture ~.